src/calculateRentalCost.js

```diff
@@ -28,7 +28,16 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  switch (true) {
+    case days >= 7:
+      return days * 40 - 50;
+
+    case days >= 3:
+      return days * 40 - 20;
+
+    default:
+      return days * 40;
+  }
```

Comment:
switch true is the same as if, so it is useless and move all magic numbers to the variables